packages/fxa-settings/src/components/AppLayout/index.tsx

```diff
@@ -6,37 +6,45 @@ import React from 'react';
 import mozLogo from 'fxa-react/images/moz-logo.svg';
 import LinkExternal from 'fxa-react/components/LinkExternal';
 import { useLocalization } from '@fluent/react';
+import Head from 'fxa-react/components/Head';
 
 type AppLayoutProps = {
+  title?: string;
   children: React.ReactNode;
 };
 
-export const AppLayout = ({ children }: AppLayoutProps) => {
+export const AppLayout = ({ title, children }: AppLayoutProps) => {
   const { l10n } = useLocalization();
   return (
-    <div className="flex min-h-screen flex-col items-center" data-testid="app">
-      <main className="mobileLandscape:flex mobileLandscape:items-center mobileLandscape:flex-1">
-        <section>
-          <div className="card">{children}</div>
-        </section>
-      </main>
-      <footer className="hidden mobileLandscape:block w-full p-8">
-        <LinkExternal
-          rel="author"
-          href="https://www.mozilla.org/about/?utm_source=firefox-accounts&amp;utm_medium=Referral"
-        >
-          <img
-            src={mozLogo}
-            alt={l10n.getString(
-              'app-footer-mozilla-logo-label',
-              null,
-              'Mozilla logo'
-            )}
-            className="w-32"
-          />
-        </LinkExternal>
-      </footer>
-    </div>
+    <>
+      <Head {...{ title }} />
```

Comment:
Moved the `Head` out of `App` and into `AppLayout` so that we can pass in the page title as a prop.

No other changes to the rest of the component, not sure why it's showing as modified.

Reply:
I've created a new ticket to add page titles to other pages created during the React conversion: [FXA-6803](https://mozilla-hub.atlassian.net/browse/FXA-6803)